bxin/ptg2hex/hex_diagnostics.ipynb

```diff
@@ -0,0 +1,607 @@
+    "print(\"Hexapod hex\", whichHex1.MTHexapodID)\n",
+    "print(hex.salinfo.index)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### test state transition. If hex is already enabled, disable then enable it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "state = await hex.evt_summaryState.aget(timeout=5)\n",
+    "print('staring with: hex state', salobj.State(state.summaryState), pd.to_datetime(state.private_sndStamp, unit='s'))\n",
+    "if state.summaryState == 2:\n",
+    "    await salobj.set_summary_state(hex, salobj.State.DISABLED) #disable hex\n",
+    "    await asyncio.sleep(3)\n",
+    "    state = await hex.evt_summaryState.aget(timeout=5)\n",
+    "    print('hex state', salobj.State(state.summaryState), pd.to_datetime(state.private_sndStamp, unit='s'))\n",
+    "\n",
+    "await salobj.set_summary_state(hex, salobj.State.ENABLED, settingsToApply=\"default\") #enable hex\n",
+    "state = await hex.evt_summaryState.aget(timeout=5)\n",
+    "print('hex state', salobj.State(state.summaryState), pd.to_datetime(state.private_sndStamp, unit='s'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#may need to wait a few seconds before event shows up in EFD\n",
+    "await asyncio.sleep(5.)\n",
+    "from lsst_efd_client import EfdClient\n",
+    "\n",
+    "client = EfdClient('ncsa_teststand_efd')\n",
+    "csc_index = 1\n",
+    "end = Time(datetime.now())\n",
+    "start = end - timedelta(hours=1)\n",
+    "dfe = await client.select_time_series('lsst.sal.MTHexapod.logevent_summaryState', '*', start.tai, end.tai, csc_index)\n",
+    "#to check messages in Kafka, go to https://lsst-kafka-0-nts-efd.ncsa.illinois.edu/\n",
+    "#we cannot get time series data from DDS. We have to query the EFD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dfe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### check the behavior when LUT is disabled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lutMode = await hex.evt_compensationMode.aget(timeout=10)\n",
+    "print(\"compsensation mode enabled?\",lutMode.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pos0 = await hex.tel_application.aget(timeout=10.)\n",
+    "print(pos0.MTHexapodID)\n",
+    "for i in range(6):\n",
+    "    print('%10.2f '%pos0.position[i], end= ' ')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to observe the motions in chronograf, consider using \"AND MTHexapodID==1\" to filter out telmetry from the other hexapod"
```

Comment:
This is a good point, but please update the comment to use hexId instead of "1"

In fact I strongly suggest changing the code to use this always, as we are likely to be running both hexapods frequently soon enough -- may as well "do it right" now and avoid massive confusion when both hexapods are busy.